DigitalLearningSolutions.Web/ViewModels/LearningPortal/RecommendedLearning/AllRecommendedLearningItemsViewModel.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.LearningPortal.RecommendedLearning
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using DigitalLearningSolutions.Data.Models.LearningResources;
+
+    public class AllRecommendedLearningItemsViewModel
+    {
+        public AllRecommendedLearningItemsViewModel(IEnumerable<RecommendedResource> resources, int selfAssessmentId)
+        {
+            RecommendedResources =
+                resources.OrderByDescending(r => r.RecommendationScore).Select(r => new SearchableRecommendedResourceViewModel(r, selfAssessmentId));

```

Comment:
Do we need the order by here? Or is this to make the javascript ordering work as expected?

Reply:
Without it, the JS ordering doesn't work properly. But that is only because I have no properly set up the JS ordering as we would expect.

We need to order based on the Recommendation Score, but we don't currently pass that through to card (only the wording for the tag). I thought that, rather than adding some hidden values to each card and setting up the SortOptions, I'd just order them in this view model first. The JS manipulation then preserves the order as it is because it doesn't have anything to sort on.

It's only a small reduction in the code necessary, and not a big time sink to implement the usual way, so I'm happy to implement that if you think it'd be better than using this workaround.

Reply:
Fair enough, I don't really thing it matters either way, so lets just leave it for now